### PR TITLE
Test software to address issue #203 for vjqa

### PR DESCRIPTION
**Pull request by DanIverson**
Two vjqa log files would be made if the tests ran over midnight.
Also links a standard name programs can use to acces the last results.

### CODE REVIEW

**Comment by DanIverson** [comment]
I did some checking and my changes are not sufficient. I will do another push.

**Comment by tesch1** [comment]
rebased onto OpenVnmrJ/master, added a commit for circleci to diff against the new file: https://github.com/tesch1/OpenVnmrJ/tree/vjqa , https://circleci.com/gh/tesch1/OpenVnmrJ/184

**Comment by DanIverson** [comment]
From the circleci 184 report, it seems my changes are working. The log file has an hour and minute in the name, the report.txt is there and since it is not making a new log file every minute, it should also cross midnight without a problem.  My guess is that the error "Too long with no output error (exceeded 10m0s) " at the bottom of the "Test OpenVnmrJ" section is because it was compiling 162 solidspack pulse sequences (the last line of the "Tests Failed" section) and that takes longer than 10 minutes.  Is there a way to increase the time limit?

**Comment by tesch1** [comment]
I dont know what happened there, usually there's a little spinner running to prevent the "no output, kill" response, haven't ever seen it die like this before.  I'm re-running it locally to see what happens.

File: src/vjqa/vjtest/maclib/vjqa [hunk 1]
```diff
@@ -1,6 +1,11 @@
 "macro vjqa"
-vvLog:$file
+vvLog('newLog'):$file
 write('reset',$file)
+// Next two lines link a standard name that external programs can
+// use to get the latest results
+substr($file,'dirname'):$dir
+substr($file,'basename'):$name
+shell('cd '+$dir+'; rm -f report.txt; ln -s '+$name+' report.txt')
```
Comment (tesch1) [inline src/vjqa/vjtest/maclib/vjqa]: Maybe double quote $name in case userdir has spaces

**Comment by tesch1** [comment]
Ran ok locally on centos6,centos7,ubuntu14, and ubuntu16... I'd say it's good. If that hang pops up again on circleci, we know where it started.

**Pull request merged**